### PR TITLE
Fixed regression test failure

### PR DESCRIPTION
**Pull request by zwyan0**
Fixed a problem that caused some regression tests of pg_rman to fail since 15.4.

### CODE REVIEW

File: sql/backup_from_standby.sh [hunk 1]
```diff
@@ -101,13 +101,13 @@ EOF
 
     # start PostgreSQL
     pg_ctl start -D ${PGDATA_PATH} -w -t 300 > /dev/null 2>&1
-	mkdir -p ${TBLSPC_PATH}/pgbench	
-	psql --no-psqlrc -p ${TEST_PGPORT} -d postgres > /dev/null 2>&1 << EOF
-CREATE TABLESPACE pgbench LOCATION '${TBLSPC_PATH}/pgbench';
-CREATE DATABASE pgbench TABLESPACE = pgbench;
-EOF
+# 	mkdir -p ${TBLSPC_PATH}/pgbench
```
Comment (mikecaat) [inline sql/backup_from_standby.sh:104]: I think we need the reason why it's commented out.

**Review by mikecaat — COMMENTED** [review]
Thanks!

File: sql/backup_from_standby.sh [hunk 2]
```diff
@@ -101,13 +101,13 @@ EOF
...
     pg_ctl start -D ${PGDATA_PATH} -w -t 300 > /dev/null 2>&1
-	mkdir -p ${TBLSPC_PATH}/pgbench	
-	psql --no-psqlrc -p ${TEST_PGPORT} -d postgres > /dev/null 2>&1 << EOF
-CREATE TABLESPACE pgbench LOCATION '${TBLSPC_PATH}/pgbench';
-CREATE DATABASE pgbench TABLESPACE = pgbench;
-EOF
+# 	mkdir -p ${TBLSPC_PATH}/pgbench	
+# 	psql --no-psqlrc -p ${TEST_PGPORT} -d postgres > /dev/null 2>&1 << EOF
+# CREATE TABLESPACE pgbench LOCATION '${TBLSPC_PATH}/pgbench';
+# CREATE DATABASE pgbench TABLESPACE = pgbench;
```
[the remark this answers is not in the document]
Comment (mikecaat) [reply sql/backup_from_standby.sh:107]: Sorry, I think it'ss better to replace to `CREATE DATABASE pgbench`

[the remark this answers is not in the document]
Comment (zwyan0) [reply sql/backup_from_standby.sh]: We don need the line. So I removed this line and CREATE DATABASE pgbench to replaced it

File: sql/backup_from_standby.sh [hunk 3]
```diff
@@ -101,13 +101,13 @@ EOF
...
+# EOF
 
-    pgbench -i -s $SCALE -p ${TEST_PGPORT} -d pgbench > ${TEST_BASE}/pgbench.log 2>&1
+    pgbench -i -s $SCALE -p ${TEST_PGPORT} -d postgres > ${TEST_BASE}/pgbench.log 2>&1
```
[the remark this answers is not in the document]
Comment (zwyan0) [reply sql/backup_from_standby.sh]: Yes I agree with you. So I revert this change.

File: sql/backup_from_standby.sh [hunk 4]
```diff
@@ -119,40 +119,32 @@ function init_catalog()
...
-EOF
+	pg_basebackup -d "dbname=postgres host=localhost port=${TEST_PGPORT}" -D ${SBYDATA_PATH}  --checkpoint=fast > /dev/null 2>&1
 
 	cp ${SBYDATA_PATH}/postgresql.conf_org ${SBYDATA_PATH}/postgresql.conf
 	cat >> ${SBYDATA_PATH}/postgresql.conf << EOF
 port = ${TEST_SBYPGPORT}
 hot_standby = on
 logging_collector = on
-wal_level = hot_standby
+wal_level = replica
```
[the remark this answers is not in the document]
Comment (zwyan0) [reply sql/backup_from_standby.sh]: I will push another commit to change it.

**Review by mikecaat — COMMENTED** [review]
Thanks!

**Review by mikecaat — APPROVED** [review]

**Comment by mikecaat** [comment]
I confirmed the change. Thanks for your works!
Please don't forget to apply them to back branches.

**Pull request merged**